cypress/e2e/with-users/machines/actions.spec.ts

```diff
@@ -1,26 +1,45 @@
 import { generateMAASURL, generateName } from "../../utils";
 
-const MACHINE_ACTIONS = [
-  "Commission",
-  "Allocate",
-  "Deploy",
-  "Release",
-  "Abort",
-  "Clone from",
-  "Power on",
-  "Power off",
-  "Test",
-  "Enter rescue mode",
-  "Exit rescue mode",
-  "Mark fixed",
-  "Mark broken",
-  "Override failed testing",
-  "Lock",
-  "Unlock",
-  "Tag",
-  "Set zone",
-  "Set pool",
-  "Delete",
+type ActionGroup = {
```

Comment:
No need to manually define the type here, you can just set the `MACHINE_ACTIONS_GROUPS` as read-only and let TS know this will never change.